### DISCUSSION TITLE
Sign in issue with CoPilot Jetbrains IntelliJ v. 1.2.22.3129 plugin

### DISCUSSION THREAD

**Discussion by ruirrib** [community/community#67189] [Copilot Conversations]
### Select Topic Area

Bug

### Body

This plugin version errors out with an error message of the type:

github copilot sign in failed reason: request signInInitiate failed with message E8520000:error:0A000152SSLroutines:unsafe legacy renogotiation disabled

Going back to version 1.2.21.2995 makes the error go away and the plugin works as expected.

**Comment by connorjs** [comment]
I also hit this issue. I believe corporate environments may have issues with OpenSSL 3. Downgrading to the previous version fixed it for me.

Did the latest version (`1.2.22.3129`) include an OpenSSL 3 upgrade?

**Comment by evadell** [comment]
Same error here in a corporate environment (MITM ssl proxy), with copilot version 1.2.22.3129. This is itellij log:

`2023-09-22 13:02:46,635 [3947301]   WARN - #com.github.copilot.lang.agent.AgentGitHubService - Failed to initiate sign in
com.github.copilot.lang.agent.SignInException: Request signInInitiate failed with message: B08B0000:error:0A000152:SSL routines:final_renegotiate:unsafe legacy renegotiation disabled:C:\Users\runneradmin\AppData\Local\Temp\pkg.24e0b2b2d51e47b9dba34c30\node\deps\openssl\openssl\ssl\statem\extensions.c:908:
, request id: 6, error code: -32603
	at com.github.copilot.lang.agent.AgentGitHubService.initiateSignIn(AgentGitHubService.java:198)
...
`
